d3a_api_client/rest_device.py

```diff
@@ -1,12 +1,13 @@
 import logging
+
```

Comment:
I'm not sure that this blank line is needed: `concurrent.futures` is part of the standard library as well I think.